pkg/control/sidecarcontrol/util_test.go

```diff
@@ -1244,6 +1244,32 @@ func TestPodMatchedSidecarSet(t *testing.T) {
 			},
 			expect: false,
 		},
+		{
```

Comment:
You need modidy the sidecarSet [webhook](https://github.com/openkruise/kruise/blob/master/pkg/webhook/sidecarset/validating/sidecarset_create_update_handler.go#L118) and [controller](https://github.com/openkruise/kruise/blob/master/pkg/controller/sidecarset/sidecarset_processor.go#L74).

And you must add [e2e](https://github.com/openkruise/kruise/blob/master/test/e2e/apps/sidecarset.go) for this scenario.

Reply:
Done, please review my latest pr, thanks